commit 83e93116882474534b8258d8d0d68f98c385aafe
Author: CI Bot <ci-bot@internal.example>
Date:   Mon Feb 3 14:22:00 2025 +0000

    Add temporary config-sync deploy credential (TODO: move to vault)

diff --git a/deploy/config-sync.conf b/deploy/config-sync.conf
index 1a2b3c4..5d6e7f8 100644
--- a/deploy/config-sync.conf
+++ b/deploy/config-sync.conf
@@ -1,2 +1,3 @@
 name = "config-sync"
+secret = "rOkuAWj51fhnDMEhU1gIVii8yRUi1d"
 enabled = true
